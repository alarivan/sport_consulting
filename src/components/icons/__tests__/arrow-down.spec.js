import React from "react"
import { render } from "@testing-library/react"
import ArrowDown from "../ArrowDown.js"

describe("Icon/arrow-down", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<ArrowDown />)

    expect(asFragment()).toMatchSnapshot()
  })
})
