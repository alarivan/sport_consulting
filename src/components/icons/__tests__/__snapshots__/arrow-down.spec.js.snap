// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon/arrow-down renders correctly 1`] = `
<DocumentFragment>
  .c0 {
  display: block;
  width: 1rem;
  height: 1rem;
}

<svg
    class="c0"
    viewBox="0 0 32 32"
  >
    <path
      d="M16 31l15-15h-9v-16h-12v16h-9z"
    />
  </svg>
</DocumentFragment>
`;
